### ISSUE TITLE
 Get.snackbar error 'overlayEntries.isEmpty': is not true.

### ISSUE THREAD

**Issue by IsmailAlamKhan** [jonataslaw/getx#821]
Gettting this assertion after snackbar close
```
[GETX] CLOSE SNACKBAR null

════════ Exception caught by animation library ═════════════════════════════════
The following assertion was thrown while notifying status listeners for AnimationController:
Assertion failed:
C:\…\snackbar\snack_route.dart:263
overlayEntries.isEmpty
is not true


When the exception was thrown, this was the stack
C:/b/s/w/ir/cache/builder/src/out/host_debug/dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 216:49  throw_
C:/b/s/w/ir/cache/builder/src/out/host_debug/dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 24:3    assertFailed
packages/get/get_navigation/src/snackbar/snack_route.dart 263:33                                                           [_handleStatusChanged]
packages/flutter/src/animation/listener_helpers.dart 199:27                                                                notifyStatusListeners
packages/flutter/src/animation/animation_controller.dart 795:7                                                             [_checkStatusChanged]
...
The AnimationController notifying status listeners was: AnimationController#cf1b4(⏮ 0.000; paused; for SnackRoute<dynamic>)
════════════════════════════════════════════════════════════════════════════════

```
code: 
```   
 double deviceWidth(BuildContext context) => context.width;
 bool isMobile(BuildContext context) => deviceWidth(context) <= 800;

Get.rawSnackbar(
      snackStyle: SnackStyle.FLOATING,
      backgroundColor: errorColor,
      borderRadius: 5,
      messageText: DefaultText(
        text: 'Hello I am a snackbar',
        style: TextStyle(
          color: Colors.white,
          fontSize: 16,
        ),
      ),
      isDismissible: true,
      maxWidth: isMobile(context)
          ?deviceWidth(context)
          : deviceWidth(context) / 2.5,
      forwardAnimationCurve: Curves.easeInOut,
      barBlur: 20,
      margin: onlyPad(
        bottom: 30,
        left: 20,
        right: 20,
      ),
    );
```
No problem with its functionality but getting the assertion.

**Comment by volgin** [comment]
Same error with Get.snackbar(...).

**Comment by yulkin2002** [comment]
I had this issue and according to an old issue logged against flutter (which I cannot find right now) it's caused by a corrupt flutter installation. What fixed it for me is to Invalidate Caches/Restart the Android studio, run flutter clean, switch to stable channel and re-install flutter.

**Comment by yulkin2002** [comment]
and here is the link: https://github.com/jonataslaw/getx/issues/748

**Comment by IsmailAlamKhan** [comment]
@yulkin2002 that did not work for me

**Comment by martin-robert-fink** [comment]
I have this issue as well.  I completely removed my flutter installation and started from scratch.  I also created a brand new flutter project.  Running dev channel Flutter 1.25.0-4.0.pre and get: ^3.17.1.  This happens on both iOS and MacOS builds.

Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel dev, 1.25.0-4.0.pre, on macOS 11.0.1 20B29 darwin-x64, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
[✓] Xcode - develop for iOS and macOS (Xcode 12.2)
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.1)
[✓] VS Code (version 1.51.1)
[✓] Connected device (3 available)

• No issues found!

**Comment by jonataslaw** [comment]
It's a Master/Dev problem. 
Fixed on 3.22.2
